torchaudio/prototype/functional/functional.py

```diff
@@ -19,12 +19,12 @@ def fftconvolve(x: torch.Tensor, y: torch.Tensor) -> torch.Tensor:
     .. properties:: Autograd TorchScript
 
     Args:
-        x (torch.Tensor): First convolution operand, with shape `(*, N)`.
-        y (torch.Tensor): Second convolution operand, with shape `(*, M)`
+        x (torch.Tensor): First convolution operand, with shape `(..., N)`.
+        y (torch.Tensor): Second convolution operand, with shape `(..., M)`
```

Comment:
this seems related to `fftconvolve`, shall we make it in a separate PR?

Reply:
it is a very small change to a docstring in response to feedback on this pr — seems like overkill to create a separate pr for it